wolox-books/src/app/screens/auth/screens/book-detail/book-detail.component.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../../../../sass/colors';
+@import '../../../../../sass/fonts';
+
+.container {
+  display: flex;
+  width: 100%;
+  max-width: 1100px;
+  background: $white;
+  padding: 20px;
+
+  .logo {
+    padding-right: 20px;
+    width: 400px;
```

Comment:
Setting a fixed width this big might break the layout  in smaller displays, I think you should change this to width 100% max width 400px